test/feature/NumericLiteral/Simple.js

```diff
@@ -83,6 +83,19 @@
     static get 0O12() {}
     static set 0O13(v) {}
   }
-  assertArrayEquals(['0', '1', '2', '6', '7', '8'], Object.keys(C.prototype));
-  assertArrayEquals(['3', '4', '5', '9', '10', '11'], Object.keys(C));
-})();
\ No newline at end of file
+
+  assert.isTrue(C.prototype.hasOwnProperty('0'));
+  assert.isTrue(C.prototype.hasOwnProperty('1'));
```

Comment:
This change reduces coverage by allowing extra properties to sneak in. Maybe use gOPNs instead of O.keys?